src/code/z_map_mark.c

```diff
@@ -63,10 +63,10 @@ void MapMark_Init(GlobalContext* globalCtx) {
     Overlay_Load(overlay->vromStart, overlay->vromEnd, overlay->vramStart, overlay->vramEnd, overlay->loadedRamAddr);
 
     sLoadedMarkDataTable = gMapMarkDataTable;
-    sLoadedMarkDataTable = (void*)(u32)(
-        (overlay->vramTable != NULL)
-            ? (void*)((u32)overlay->vramTable - (s32)((u32)overlay->vramStart - (u32)overlay->loadedRamAddr))
-            : NULL);
+    sLoadedMarkDataTable = (void*)(u32)((overlay->vramTable != NULL)
+                                            ? (void*)((u32)overlay->vramTable -
+                                                      (s32)((u32)overlay->vramStart - (u32)overlay->loadedRamAddr))
+                                            : NULL);
```

Comment:
the format changes are still here. id probably just revert the files with unwanted format changes and we can do more research on the clang differences separately